src/forms/fields/SelfAssessmentQuestion.php

```diff
@@ -112,8 +112,11 @@ public function getFormField()
 
     public function getIsLastQuestion()
     {
-        $fields = $this->Parent()->Fields()->column('ID');
-        return array_search($this->ID, $fields) === count($fields) - 1;
+        if ($this->Parent()) {
+            $fields = $this->Parent()->Fields()->column('ID');
```

Comment:
@lukereative You may want to check that the array is not empty either, otherwise array_search is going to throw an error.